src/Shell/Task/TestTask.php

```diff
@@ -246,11 +248,12 @@ public function bake($type, $className)
         $this->BakeTemplate->set(compact(
             'subject',
             'className',
+            'properties',
             'methods',
             'type',
             'fullClassName',
             'mock',
-            'realType',
```

Comment:
@markstory  `$realType` was defiend quite some time ago, but it has been removed in adb7f6025f4dae735c88f576a1b2806b5911a2f3 Removing the key shouldn't change the current behavior.